src/main/resources/application.yaml

```diff
@@ -1025,6 +1025,9 @@ notification:
         docmosisId: TB-SCS-GNO-WEL-00774.docx
       appointee:
         docmosisId: TB-SCS-GNO-WEL-00774.docx
+    updateOtherPartyData:
```

Comment:
There are few test class that testing this config and new template. You need to add this new template there.

Reply:
added the test